source/getting-started/js-primer.md

```diff
@@ -1,74 +1,99 @@
 not every feature of the JavaScript language may be familiar to the developer.
 
-In this guide we will be covering some JavaScript features,
-and how they are used in Ember applications.
+This guide will cover some JavaScript features, and how to use them in Ember applications.
+To see a full set of features added in ES2015, check the [Mozilla Developer Network Docs](https://developer.mozilla.org/en/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla).
 
 ## Variable declarations
 
-A variable declaration, also called binding, is when you assign a value to a variable name.
-An example of declaring a variable containing the number 42 is like so:
+A variable declaration is when you assign a variable name to a location in memory.
+This creates a place in memory to store data and allows lookup for that data by name.
+The following code declares a variable called `myNumber` and stores the number 42:
 
 ```javascript
 var myNumber = 42;
 ```
 
-JavaScript initially had two ways to declare variables, globally and `var`.
-With the release of ES2015, `const` and `let` were introduced.
-We will go through the different ways to declare a variable,
-also called bindings because they *bind* a value to a variable name,
-and why modern JavaScript tends to prefer `const` and `let`.
+The values of most Javascript variables can be changed using the `=` operator in code.
+However, changing the value of an existing variable should not use the `var`, `let` or `const` keywords since using the keyword will try to declare a new variable with the same name.
+The following code declares the variable `text` and changes the value.
+
+```javascript
+var text = "Hello World";
+
+text = "Hello Browser";
+```
+
+Before ES2015, JavaScript had to one way to declare variables using the `var` keyword.
+ES2015 introduced two new variable keywords: `const` and `let`.
+Each of these keywords has a different way of declaring variables.
+This guide will review the differences between `var`, `const` and `let`; and will discuss when to choose between them.
 
 ### `var`
 
-Variable declarations using `var` exist in the entire body of the function where they are declared.
-This is called function-scoping, the existence of the `var` is scoped to the function.
-If you try to access a `var` outside of the function it is declared,
-you will get an error that the variable is not defined.
+Variables declared using the `var` keyword only exist within that function.
+This is called *function-scoping*.
+Trying to access a variable declared with `var` outside of its function will throw an error.
+This is because the variable is not declared in the current scope.
 
-For our example, we will declare a `var` named `name`.
-We will try to access it both inside the function and outside,
-and see the results we get:
+The following code declares a variable using  `var` inside of a function.
+Then, the variable is logged both inside and outside of that function:
```

Comment:
I'm a bit 🤔 about this sentence, I think there should be some mention that it "tries" to log the variable since one of the instances will lead to an error

Reply:
Maybe split into two sentences? It logs inside the function. It tries to log outside but triggers an error.